frontend/jupyter/src/app/pages/notebook-page/notebook-page.component.spec.ts

```diff
@@ -36,7 +36,7 @@ const ActionsServiceStub: Partial<ActionsService> = {
 };
 const NamespaceServiceStub: Partial<NamespaceService> = {
   updateSelectedNamespace: () => {},
-  getSelectedNamespace2: () => of(),
+  getSelectedNamespace: () => of(),
```

Comment:
Did you really rename their code?

Reply:
No, we just use `getSelectedNamespace` instead of `getSelectedNamespace2`. Its two seperate functions